bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/HelpScreen.kt

```diff
@@ -0,0 +1,59 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import org.hamcrest.CoreMatchers.allOf
+
+class HelpScreen {
+    fun clickTipsForBestResults(): HelpScreen {
+        onView(
+            allOf(
+                withId(net.gini.android.capture.R.id.gc_help_item_title),
+                withText("Tips for best results from photos"),
+                isDescendantOfA(withId(net.gini.android.capture.R.id.gc_help_items))
+            )
+        )
+            .check(matches(withText("Tips for best results from photos")))
```

Comment:
[Clarification] Is this check needed? 
It seems to me like the test not fail anyway if the view returned by `onView` does not exist? 
I could be missing something though 🤔

Reply:
I would want to know how can be make it more of a valuable test, let's chat on this suggestion. 🙂